package.json

```diff
@@ -6,37 +6,25 @@
     "files": [
         "src/",
         "pages/",
-        "doczrc.js",
         "tsconfig.json",
         "package.json"
     ],
     "scripts": {
-        "dev": "docz dev",
-        "build": "docz build",
-        "serve": "docz serve",
+        "build": "mdbook build -o",
+        "serve": "mdbook serve -o",
+        "watch": "mdbook watch -o",
         "spellcheck": "mdspell './pages/**/*.md' './*.md' '!./LICENSE.md' --en-us --ignore-numbers --report",
         "spellfix": "mdspell './pages/**/*.md' './*.md' '!./LICENSE.md' --en-us --ignore-numbers",
         "lint-md": "remark --frail ./pages/**/*.md ./pages/*.md",
         "lint": "npm run lint-md && npm run spellcheck"
     },
```

Comment:
are we going to retire npm/docz @wilwade  @mehtaishita

Reply:
mostly linting is used from npm perspective so it can be removed with how we can lint with cargo and get rid of docz completely